tests/ItemTest.php

```diff
@@ -67,4 +68,29 @@ public function testIsHit2()
 
         $this->assertFalse($item->isHit());
     }
+
+    public function testExpiresAt()
+    {
+        $item = new Item("test");
+
+        $this->assertInstanceOf(Item::class, $item->expiresAt(new \DateTime()));
```

Comment:
Shouldn't this be part of the `expiresAfterDataProvider`?

Reply:
I think that method is different from ```expiresAfter```.
This is the ```expiresAt```.

Reply:
Ah sorry, I missed that :+1: